test/test.ml

```diff
@@ -85,20 +101,35 @@ let bad_test f file () =
 
 let excl_test file () =
   Alcotest.(check bool) "good test" true true
+module P4 = Types 
+let pp_test () =
+  Alcotest.(check bool) "printing test" true Printingtests.test
+
+let example_path l =
+  let root = Filename.concat ".." "examples" in
+  List.fold_left l ~init:root ~f:Filename.concat
 
-let () =
+let () = 
+  (* Format.printf "%s" (get_name ["../examples"] (example_path ["checker_tests"; "good"; "coree.p4"])); 
+     Format.printf "%s" "----------";
+     Format.printf "%s" ((Parse.parse_string (get_name ["../examples"] (example_path ["checker_tests"; "good"; "coree.p4"]))) |> Prettypp.format_program |> to_string);   *)
```

Comment:
Can you delete this commented out code and get rid of coree.p4? It's a duplicate of pragma-string.p4 in the same directory anyway.